frontend/src/pages/Dashboard/Analytics.tsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
+import AnalyticsTable from '../../components/AnalyticsTable/AnalyticsTable';
 import TabSwitcher, { Tab } from '../../components/TabSwitcher/TabSwitcher';
 
 const Analytics = () => (
    <div>
       <TabSwitcher>
```

Comment:
I'm not sure if this should be fixed here or in accessibility-fixes, but this tabswitcher is not accessible. Because it uses <h1>, it's not focusable by keyboard/screen reader will treat it like normal text instead of something interactable. This means if I try to use the keyboard to navigate to here, I can't reach it.

Reply:
probably can do in the accessibility branch